### DISCUSSION TITLE
Tablet Support

### DISCUSSION THREAD

**Discussion by xo0ps** [mobile-dev-inc/Maestro#1092] [Q&A]
Hi,

I could see that you support 1 Android emulator and 1 iOS simulator on [Maestro Cloud](https://cloud.mobile.dev/reference/device-configuration). For Android it is possible to choose from 4 API levels.
Is there a possibility to support Tablets? And is it possible to include API 28 as well?

**Comment by axelniklasson** [comment]
Hi there @xo0ps! Could you please file these feature requests [here](https://github.com/mobile-dev-inc/maestro/issues/new?assignees=&labels=enhancement&projects=&template=feature_request.md&title=%5BFeature+Request%5D)? Thanks!
